templates/branding/docbox.html.ep

```diff
@@ -0,0 +1,5 @@
+DOCBOX
```

Comment:
and I think this should live in branding/plain - which is where the unbranded files live. and then you don't need the one liner templates